datafusion/datasource-arrow/src/file_format.rs

```diff
@@ -175,10 +182,39 @@ impl FileFormat for ArrowFormat {
 
     async fn create_physical_plan(
         &self,
-        _state: &dyn Session,
+        state: &dyn Session,
         conf: FileScanConfig,
     ) -> Result<Arc<dyn ExecutionPlan>> {
-        let source = Arc::new(ArrowSource::default());
+        let is_stream_format = if let Some(first_group) = conf.file_groups.first() {
```

Comment:
Maybe worth pulling this out into a helper method that's easier to test. Also then this method reads a bit cleaner, with just a is_stream_format() check as opposed to this block of logic.

Reply:
fixed! I opted to go with a positive check for whether it's in the arrow file format. Future steps that perform the actual parsing of the file should catch if it's not in the arrow stream format either.